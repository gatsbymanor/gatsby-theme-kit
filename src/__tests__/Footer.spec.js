import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

describe('<Footer />', () => {
  const component = renderer.create(<Footer />).toJSON();

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});