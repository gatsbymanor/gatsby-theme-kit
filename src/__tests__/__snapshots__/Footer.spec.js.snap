// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<footer
  id="footer"
>
  <section>
    <h2>
      Aliquam sed mauris
    </h2>
    <p>
      Sed lorem ipsum dolor sit amet et nullam consequat feugiat consequat magna adipiscing tempus etiam dolore veroeros. eget dapibus mauris. Cras aliquet, nisl ut viverra sollicitudin, ligula erat egestas velit, vitae tincidunt odio.
    </p>
    <ul
      className="actions"
    >
      <li>
        <a
          className="button"
          href="generic.html"
        >
          Learn More
        </a>
      </li>
    </ul>
  </section>
  <section>
    <h2>
      Etiam feugiat
    </h2>
    <dl
      className="alt"
    >
      <dt>
        Address
      </dt>
      <dd>
        1234 Somewhere Road • Nashville, TN 00000 • USA
      </dd>
      <dt>
        Phone
      </dt>
      <dd>
        (000) 000-0000 x 0000
      </dd>
      <dt>
        Email
      </dt>
      <dd>
        <a
          href="#"
        >
          information@untitled.tld
        </a>
      </dd>
    </dl>
    <ul
      className="icons"
    >
      <li>
        <a
          className="icon fa-twitter alt"
          href="#"
        >
          <span
            className="label"
          >
            Twitter
          </span>
        </a>
      </li>
      <li>
        <a
          className="icon fa-facebook alt"
          href="#"
        >
          <span
            className="label"
          >
            Facebook
          </span>
        </a>
      </li>
      <li>
        <a
          className="icon fa-instagram alt"
          href="#"
        >
          <span
            className="label"
          >
            Instagram
          </span>
        </a>
      </li>
      <li>
        <a
          className="icon fa-github alt"
          href="#"
        >
          <span
            className="label"
          >
            GitHub
          </span>
        </a>
      </li>
      <li>
        <a
          className="icon fa-dribbble alt"
          href="#"
        >
          <span
            className="label"
          >
            Dribbble
          </span>
        </a>
      </li>
    </ul>
  </section>
  <p
    className="copyright"
  >
    © Untitled. Design: 
    <a
      href="https://html5up.net"
    >
      HTML5 UP
    </a>
    .
  </p>
</footer>
`;
